### ISSUE TITLE
Bigger buttons for tablets

### ISSUE THREAD

**Issue by lucifurious** [irssiconnectbot/irssiconnectbot#4]
OK, it just may be me, but trying to press the Ctl or Escape buttons is always a dreaded task. they are so small on my screen. it takes three or four tries. I have no trackball (nookcolor) so it is my only option.

please add a "big button" preference.

**Comment by lizsugar** [comment]
How much bigger are we talking here? I've got some ideas I'm playing with.

**Comment by lucifurious** [comment]
At least twice the current size please. Just talking about the ctrl, esc,
etc. Not keyboard.

I have a 7" Nook and I have to try two or three times to hit ctrl.
On Jan 7, 2012 2:10 PM, "Anthony Lademan" <
reply@reply.github.com>
wrote:

**Comment by lizsugar** [comment]
OK, yeah, the test images I made were 200% size.  I'm not going to be able to work on it tonight, but let me see what I can do over the next couple of days.  

Ideally this should target all tablet users, but for now I'll focus on your device.  Do you know the resolution and DPI at which your device runs?  I want to set up an emulator to match that so we're looking at roughly the same thing.  Thanks!

**Comment by lucifurious** [comment]
Mine is 1024x600 on a 7" screen. I have my DPI set to 180 but the default
with CM7 is 160 or so.

Can't u query the display size and make the buttons proportional? I've done
some Android work...
On Jan 7, 2012 6:22 PM, "Anthony Lademan" <
reply@reply.github.com>
wrote:

**Comment by lizsugar** [comment]
Yeah, you're right, that would probably be easier and better.  I haven't done much in that regard, though, so it will require some research.
